src/test/java/marquez/db/models/NamespaceRowTest.java

```diff
@@ -26,7 +26,7 @@
   private static final Instant CREATED_AT = Instant.now();
   private static final Instant UPDATED_AT = Instant.now();
   private static final String NAME = "test_namespace";
-  private static final String CURRENT_OWNER_NAME = "test owner";
+  private static final String CURRENT_OWNER_NAME = "test connectionUrl";
```

Comment:
Same comment around renaming

Reply:
:( so many of these

Reply:
Please see PRs #394, #387, and #384, where this is fixed.